### PR TITLE
Manpage added

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: doc/hadfs-exe.1 [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+.EX
+$ \fBmkdir\fR /home/user/ad-settings
+$ \fBkinit\fR administrator@DOMAIN.ALT
+$ \fBhadfs-exe\fR /home/user/ad-settings dc0.domain.alt
+.EE
+.SH SEE ALSO
+.BR kinit (1),
+.BR <https://www.altlinux.org/ActiveDirectory/ADFS>
+.SH AUTHOR
+This software is written by Sergei Bubnov.
```
Comment (omgbebebe) [inline doc/hadfs-exe.1]: typo in name?:)

File: doc/hadfs-exe.1 [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+.SH REPORTING BUGS
+Report bugs to: <https://github.com/omgbebebe/hadfs>
```
Comment (omgbebebe) [inline doc/hadfs-exe.1]: maybe it should point to github.com/altlinuxteam/hasdf?

File: doc/hadfs-exe.1 [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+.SH COPYRIGHT
+Copyright Sergey Bubnov (c) 2019
```
Comment (omgbebebe) [inline doc/hadfs-exe.1:58]: according to this PR there must be also the `NIR-Ginko` with `Sergey Bubnov`

File: doc/hadfs-exe.1 (hunk 1 above)
Comment (ghost) [reply doc/hadfs-exe.1]: Fixed.

File: doc/hadfs-exe.1 (hunk 2 above)
Comment (ghost) [reply doc/hadfs-exe.1]: Fixed.

File: doc/hadfs-exe.1 (hunk 3 above)
Comment (ghost) [reply doc/hadfs-exe.1:58]: Fixed. Also `AUTHORS.md` added and `ChangeLog.md` updated.

**Pull request merged**